src/assets/css/app.css

```diff
@@ -717,6 +717,37 @@ input:checked + .slider:before {
   background-color: #673ab7;
 }
 
+/* graphed Video Bubbles
```

Comment:
Don't know if its overhead or decrease loading time. Maybe we can put this in a different CSS file? To keep things seperate?